[META] docs(README): proposed standards for RFC process (#7)
stjudecloud/rfcs: README.md

[![Build Status](https://travis-ci.org/stjudecloud/rfcs.svg?branch=master)](https://travis-ci.org/stjudecloud/rfcs) [![Join the chat at https://gitter.im/stjudecloud/rfcs](https://badges.gitter.im/stjudecloud/rfcs.svg)](https://gitter.im/stjudecloud/rfcs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

This repository contains all Request for Comments (or [rfcs][rfcs]) for the St. Jude Cloud project. Currently, RFCs on the St. Jude Cloud project are focused on outlining changes to the genomic analysis pipelines. However, we may expand to other areas over time.

## Process and Evaluation

> First, the initiation of an RFC for for the St. Jude Cloud project is currently limited to members of the St. Jude Cloud team. If you are a member of our community and have an idea you would like us to consider (or if you would like to propose your own RFC), please [contact us][contact] before investing a large amount of time in your proposal.

Reviewer: ```suggestion
Initiation of an RFC for the St. Jude Cloud project is currently limited to members of the St. Jude Cloud team. If you are a member of our community and have an idea you would like us to consider (or if you would like to propose your own RFC), please [contact us][contact] before investing a large amount of time in your proposal.
```

### Overview

All RFCs go through a three-stage process from draft to adoption:

1. An initial draft is constructed by the author and discussed internally with members of the core St. Jude Cloud team. 
   * Titles of pull requests in this phase are typical prefixed with `[WIP]` and should not be considered mature enough to accept comments from the community. 
   * The amount of time the RFC spends in this state is variable depending on the scope of the proposal.
2. The RFC opens for discussion from users and stakeholders within the St. Jude community. 
   * The beginning of this phase will be indicated by a comment on the pull request from the author.
   * The RFC will remain in this phase for **1 week**.
3. The RFC opens for discussion from the broader community.
   * The beginning of this phase will be indicated by a comment on the pull request from the author.
   * The RFC will remain in this phase for **2 weeks**.

> Each phase of the process brings about further fleshing out of details and hardening of the proposal.

Reviewer: ```suggestion
Each phase of the process brings about further refinement of details and hardening of the proposal.
```

### Evaluation and acceptance

> RFCs must have majority support within the St. Jude Cloud team before being adopted. In practice, most RFCs already have a significant amount of internal support before being drafted, so the rejection of an RFC will be exceedingly rare. The intent of making these discussions is mostly geared towards gaining feedback from the community. 

Reviewer: ```suggestion
RFCs must have majority support within the St. Jude Cloud team before being adopted. In practice, most RFCs already have a significant amount of internal support before being drafted, so the rejection of an RFC will be exceedingly rare. The intent of these discussions is mostly geared towards gaining feedback from the community. 
```
